### ISSUE TITLE
Update confusing usage instructions

### ISSUE THREAD

**Issue by Eloston** [ungoogled-software/ungoogled-chromium-binaries#9]
* Update platform_ini_generator.py to use argparse.
* Update README instructions to be less confusing.

Inspired by https://github.com/Eloston/ungoogled-chromium/issues/347#issuecomment-372065025

**Comment by brokoli-mate** [comment]
can you make examples of commands please?

**Comment by brokoli-mate** [comment]
how do i pull in new changes to my fork?

**Comment by brokoli-mate** [comment]
Thanks a lot! I'll try it after compilling 65 version.

**Comment by brokoli-mate** [comment]
What should I write in place of /path/to/binaries/ ? I've tried https://github.com/brokoli-mate/ungoogled-chromium-binaries/releases/download/64.0.3282.186-1/ and https://github.com/brokoli-mate/ungoogled-chromium-binaries/releases and https://github.com/brokoli-mate/ungoogled-chromium-binaries/releases/tag/64.0.3282.186-1 and even direct links , but get FileNotFoundError: [Errno 2] No such file or directory

**Comment by Eloston** [comment]
30a1123db297c3e6494e44707a82a454e2a48f9d

**Comment by brokoli-mate** [comment]
```
./utilities/site_generator.py
Traceback (most recent call last):
  File "./utilities/site_generator.py", line 21, in <module>
    import markdown # Python-Markdown: https://github.com/waylan/Python-Markdown
ModuleNotFoundError: No module named 'markdown'
```
But markdown is installed

**Comment by Eloston** [comment]
4e3bb33caa99604372983df45746cd8ce690dcef

**Comment by brokoli-mate** [comment]
The problem was that i installed markdown through pip. After installing python3-markdown through apt it worked. 
Also get this, while running `git commit`:
```
*** Please tell me who you are.

Run

  git config --global user.email "you@example.com"
  git config --global user.name "Your Name"

to set your account's default identity.

```
So, maybe you should add this to readme also.

**Comment by Eloston** [comment]
I addressed the markdown module issue in c741a37535d8d1594a2b42f2e48a45a459d923bb. The git problem is straight-forward to solve.
